src/shared/components/challenge-listing/index.jsx

```diff
@@ -85,6 +85,12 @@ export default function ChallengeListing(props) {
     );
   }
 
+  /* eslint-env browser */
+  if (document) {
```

Comment:
@daft300punk To check, whether the code is executed at server or client side, please always use `isClientSide()` or `isServerSide()` from https://github.com/topcoder-platform/community-app/blob/develop/src/shared/utils/isomorphy.js

Also, placing this code into render method, I expect, will have side effects: ReactJS is free to re-render a component in many cases, not only when we enter the page, thus this code, I believe, will scroll the page to top in some cases when we don't need it. I guess, we should put it into `onClick` handler of the `View more challenges` button. Also, inside that handler you don't have to check for client side, as click handlers are always invoked at client-side only.